package.json

```diff
@@ -287,6 +286,7 @@
     "@types/joi": "^13.4.2",
     "@types/jquery": "^3.3.6",
     "@types/js-yaml": "^3.11.1",
+    "@types/json5": "^0.0.30",
```

Comment:
Unrelated change automatically done by `yarn`?

Reply:
Yup, likely just wasn't properly alphabetized before?

Reply:
might as well leave it in, it'll just end up as an unrelated change in some other PR :D